### DISCUSSION TITLE
[Bug report] windows系统乱码

### DISCUSSION THREAD

**Discussion by CaiWenlie** [vuepress/core#1395] [Q&A] [answered]
帮助文档里面的操作
```
mkdir docs
echo '# Hello VuePress' > docs/README.md
```
在windows系统下会有乱码

**Comment by hmyjyghh** [comment]
我也遇到该问题

**Reply by hmyjyghh** [reply]
![Snipaste_2024-08-29_14-13-50](https://github.com/user-attachments/assets/fa68425a-3266-4028-a8cf-e8e379102ac2)

**Comment by Mister-Hope** [comment, marked as answer]
请使用 Powershell 之类的终端确保你注入的字符串是 utf8

**Reply by hmyjyghh** [reply]
跟命令行工具有关系吗

**Reply by hmyjyghh** [reply]
用了 powershell 也不行  
![image](https://github.com/user-attachments/assets/8bf746ab-2b13-4fa7-b7ab-de72b0713566)
